### PR TITLE
fixing span tag syntax

### PR DESCRIPTION
**Pull request by srijitamukherjee**
**What this PR does / why we need it**:

rectifying syntactical error of span tag

**Which issue(s) this PR fixes** 

fixes #=4310

**Verification steps** 

Follow this flow - Audience > Applications > Listing
Click the multi selector.
 You'll see a space between the number of applications and the term applications, which was missing before

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by thomasmaas** [comment]
@macejmic shouldn't have been merged

File: app/views/buyers/applications/_listing.html.slim [hunk 1]
```diff
@@ -2,7 +2,7 @@ div id="bulk-operations" style="display: none;"
   h3 Bulk operations
   p
     ' You have selected
-    span class="count"
+    span.count>
     | applications and you can make following operations with them:
```
Comment (thomasmaas) [inline app/views/buyers/applications/_listing.html.slim]: text will be outside of span tag

Comment (thomasmaas) [reply app/views/buyers/applications/_listing.html.slim]: sorry, it's supposed to be outside of span tag

**Comment by thomasmaas** [comment]
@macejmic ignore me, it's fine.

File: app/views/buyers/applications/_listing.html.slim (hunk 1 above)
Comment (macejmic) [reply app/views/buyers/applications/_listing.html.slim]: that's desired.

Comment (thomasmaas) [reply app/views/buyers/applications/_listing.html.slim]: yes, that's what i saw after my first comment ;-0

**Pull request merged**